static/app/components/discover/transactionsList.tsx

```diff
@@ -413,6 +421,7 @@ class _TransactionsList extends Component<Props> {
               numSamples: tableData?.data?.length ?? null,
               supportsInvestigationRule: this.props.supportsInvestigationRule,
               cursor,
+              view: this.props.domainViewFilters?.view,
```

Comment:
nit: should we add `domainViewFilters` to the existing props destructuring above?

Reply:
Yeah, good idea, it would be more consistent